src/message-manager/AbstractMessageManager.ts

```diff
@@ -210,7 +210,7 @@ export abstract class AbstractMessageManager<
    * @returns - Promise resolving to the messageParams with the metamaskId property removed
    */
   approveMessage(messageParams: PM): Promise<P> {
-    this.setMessageStatusApproved(messageParams.metamaskId);
+    this.setMessageStatusApproved(messageParams.metamaskId!);
```

Comment:
Is this always defined :thinking: 

If we're not sure, perhaps we should ignore this problem for now instead of asserting that it is defined.

Reply:
Updated to use a `ts-ignore` comment